apps/hash-api/src/graph/ontology/primitive/data-type.ts

```diff
@@ -8,6 +8,7 @@ import type {
   ModifyRelationshipOperation,
   OntologyTemporalMetadata,
   UnarchiveDataTypeParams,
+  UserOntologyEditionProvenanceMetadata,
```

Comment:
We don't really use `edition` to refer to ontology versions anywhere else... is it called this so that the structural query path is the same? Fine to leave it consistent if it has benefits in querying. If it was just naming I would probably go for `Version`

Reply:
I went for `edition` as we had `editionCreatedById` and `editionArchivedById` as query path for ontology previously. Also, `edition` here refers to the temporal axes of ontology types as `editionCreatedById` will also be set after unarchival. Each version have one or more editions, depending if it was archived or not.

Reply:
Fair enough, makes sense!